x-pack/plugins/apm/public/components/app/Settings/ApmIndices/index.tsx

```diff
@@ -225,17 +231,30 @@ export function ApmIndices() {
                   </EuiButtonEmpty>
                 </EuiFlexItem>
                 <EuiFlexItem grow={false}>
-                  <EuiButton
-                    fill
-                    onClick={handleApplyChangesEvent}
-                    isLoading={isSaving}
-                    disabled={status !== 'success'}
+                  <EuiToolTip
+                    content={
+                      !canSave &&
+                      i18n.translate(
+                        'xpack.apm.settings.apmIndices.noPermissionTooltipLabel',
+                        {
+                          defaultMessage:
+                            "Your user role doesn't have permissions to change APM indices",
```

Comment:
@formgeist or @bmorelli25 any thoughts about this error message?

Reply:
Looks good to me 👍